projects/blackpanther/index.js

```diff
@@ -32,8 +34,7 @@ async function farm2Tvl(chain, contract, api) {
 }
 
 module.exports = {
-  timetravel: false,
```

Comment:
why did you remove timetravel?

Reply:
Not sure what this parameter does, let me add it back